website/content/docs/security/acl/acl-system.mdx

```diff
@@ -1,429 +1,90 @@
+
+Refer to the following topics for details about tokens:
+
+- [Tokens](/docs/security/acl/acl-tokens)
+- [ACL token command line](/commands/acl/token)
+- [ACL tokens API](/api-docs/acl/tokens)
+
+## Policies
+
+An ACL policy is a set of rules grant or deny access to resources in the network.
+The person responsible for administrating ACLs in your organization will assemble and create policies and link them to tokens.
+Tokens are the artifacts distributed to users so that they can be implemented.
+In addition to the rules that authenticate access to services, several attributes may be assigned policies that determine their scope.
+
+Refer to the following topics for details about policies:
+
+- [Policies](/docs/security/acl/policies)
+- [ACL policy command line](/commands/acl/policy)
+- [ACL policy API](/api-docs/acl/policies)
+
+## Roles
+
+A role is a collection of policies that your ACL administrator can link to a token.
+They enable you to reuse policies by decoupling the policies from the token distributed to team members.
+Instead, the token is linked to the role, which is able to hold several policies that can be updated asynchronously without distributing new tokens to users.
+As a result, roles can provide a more convenient authentication infrastrcture than creating unique policies and tokens for each requester.
 
-All of these tokens except the `initial_management` token can all be introduced or updated via the [/v1/agent/token API](/api/agent#update-acl-tokens).
+Refer to the [Roles](/docs/security/acl/acl-roles) topic for additional information.
 
-In Consul 1.4 - 1.10, the following special tokens were known by different names:
-
-| New Name (1.11+)                                                                     | Old Name (1.4 - 1.10)                                                    |
-| ------------------------------------------------------------------------------------ | ------------------------------------------------------------------------ |
-| [`acl.tokens.agent_recovery`](/docs/agent/options#acl_tokens_agent_recovery)         | [`acl.tokens.agent_master`](/docs/agent/options#acl_tokens_agent_master) |
-| [`acl.tokens.initial_management`](/docs/agent/options#acl_tokens_initial_management) | [`acl.tokens.master`](/docs/agent/options#acl_tokens_master)             |
-
-#### ACL Agent Recovery Token
-
-Since the [`acl.tokens.agent_recovery`](/docs/agent/options#acl_tokens_agent_recovery) is designed to be used when the Consul servers are not available, its policy is managed locally on the agent and does not need to have a token defined on the Consul servers via the ACL API. Once set, it implicitly has the following policy associated with it
-
-In Consul 1.4 - 1.10, this was called the `agent_master` token. It was renamed to `agent_recovery` token in Consul 1.11.
+## Service Identities
```

Comment:
I think it might be better to remove "Service Identities" and "Node identities" from this overview page.  These two things are convenience policy templates (not roles). They are much less central to the ACL system (when compared to tokens, policies, and roles), and by including them on this overview I think we might be distracting from the core conceptions.

I think introducing them on the policies page might be better.